sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/util/common/Counter.java

```diff
@@ -164,10 +192,18 @@
    * @throws IllegalArgumentException if the aggregation kind is not supported
    */
   @SuppressWarnings("unused")
-  private static Counter<String> strings(String name, AggregationKind kind) {
+  private static Counter<String> strings(CounterNameAndMetadata name, AggregationKind kind) {
     return new StringCounter(name, kind);
   }
 
+  /**
+   * Constructs a new {@code Counter<String>} with an unstructured name.
+   */
+  @SuppressWarnings("unused")
+  private static Counter<String> strings(String name, AggregationKind kind) {
```

Comment:
Why do we have these unused private static methods? Why not just get rid of them?